app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -129,5 +132,21 @@ class MusicsListFragment : Fragment() {
 
     }
 
+    private fun provideMusic() {
+
+        musicListAdapter.onItemClick = {
+            val bundle = Bundle()
+
+            bundle.putParcelable(ConstValues.musicInfoBundleKey, it)
+
+            _binding!!.root.findNavController()
```

Comment:
baraye find kardan vaghean niaz be view hast?
ro khode fragment fek konam beshe